modules/apps_api/spec/requests/v0/application_directory_request_spec.rb

```diff
@@ -140,4 +140,14 @@
       end
     end
   end
+
+  describe '#get /services/apps/v0/directory/:page' do
+    context 'when paginating' do
+      it 'returns paginated apps' do
+        get '/services/apps/v0/directory?page=1'
+        body = JSON.parse(response.body)
+        expect(body['data'].length).to be <= 10
+      end
+    end
+  end
```

Comment:
Looked back at the jira ticket and looks like this may have been touched on already that we don't have the sample data to do this right now. Feel free to mark this comment "resolved" if this is the case.